tools/install_base.sh

```diff
@@ -523,8 +523,11 @@ if [[ "$OLD_PADIR" == "" ]]; then
 
     for file in $( find $PA_ROOT/default/dist/war -name 'application.properties' ); do
         sed -e "s/$(escape_lhs_sed http://localhost:8080)/$(escape_rhs_sed ${PROTOCOL}://localhost:${PORT})/g" -i "$file"
+        # for cloud_automation_service
+        sed -e "s/^server\.port=.*/server.port=$PORT/g" -i "$PA_ROOT/config/web/settings.ini" -i "$file"
```

Comment:
I don't understand this line as there is already    

sed -e "s/^web\.http\.port=.*/web.http.port=$PORT/g" -i "$PA_ROOT/default/config/web/settings.ini"
and
sed -e "s/^web\.https\.port=.*/web.https.port=$PORT/g" -i "$PA_ROOT/default/config/web/settings.ini"

Reply:
you are right, the aim was just to update cloud-automation-service config file for
server.port=8443
I am commiting the update for this line